packages/ui-components/src/components/Badge/Badge.types.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { KnownIcons } from "../Icon/Icon.component"
+
+// To Do: Externalise? Similar is used in Message component
+export type VariantType = "default" | "info" | "success" | "warning" | "danger" | "error" | "critical"
+export interface BadgeProps {
```

Comment:
So you put types separately from the file, everywhere we do other way around. I know it is a best practises what you do but it is really small component, I would suggest to keep it in one file, and when it is a complicated type we could separated.